app/views/journals/show.html.slim

```diff
@@ -3,13 +3,13 @@ h1= t_title(:show, Journal)
 .row
   .col-xs-12
     p
-      strong= "#{t_attr(:subject, Journal)}: "
```

Comment:
Are you sure these can be deleted?
These views aren't always called from within journals_controller...